.gitignore

```diff
@@ -2,3 +2,4 @@
 flymake_*
 *.swp
 *.go~
+.idea/
```

Comment:
I think you can remove the IDE metadata. Add to .gitignore perhaps.

Reply:
??

Reply:
Haha. Sorry. :-) My bad.